src/services/coreBackService.js

```diff
@@ -23,11 +25,13 @@ function postAction(req, action) {
 }
 
 function postSessionInitialise(req, authParams) {
+  const logger = req.log;
   return axiosInstance.post(API_SESSION_INITIALISE, authParams, {
     headers: {
       "ip-address": req.session.ipAddress,
       "feature-set": req.session.featureSet,
     },
+    logger,
```

Comment:
What is the need to pass in the logger here but not in the other axios calls?

Reply:
Think I just missed this when refactoring